### PR TITLE
Add Content Library Item support

### PR DESCRIPTION
**Pull request by adambarreiro**
This PR:

* Adds a **new Resource:** `vcd_tm_content_library_item` to manage Content Library Items
* Adds a **new Data Source:** `vcd_tm_content_library_item` to read Content Library Items

Resource update is not supported yet.

### CODE REVIEW

File: website/docs/d/tm_content_library_item.html.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Foundation Tenant Manager: vcd_tm_content_library_item"
+sidebar_current: "docs-vcd-data-source-tm-content-library-item"
+description: |-
+  Provides a VMware Cloud Foundation Tenant Manager Content Library Item data source. This can be used to read Content Library Items.
+---
+
+# vcd\_content\_library\_item
```
Comment (Didainius) [inline website/docs/d/tm_content_library_item.html.markdown:9]:
```suggestion
# vcd\_tm\_content\_library\_item
```

File: vcd/resource_vcd_tm_content_library_item.go [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true, // TODO: TM: Update not supported yet
+				Description: fmt.Sprintf("The description of the %s", labelTmContentLibraryItem),
+			},
+			"content_library_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true, // TODO: TM: Update not supported yet
```
Comment (Didainius) [inline vcd/resource_vcd_tm_content_library_item.go:43]: NIT. About the comment - technically, I don't think this will support update anyway

**Review by Didainius — APPROVED** [review]
Missing `vcd.erb` (index for docs).

I have run automated tests and manually tested

File: vcd/resource_vcd_tm_content_library_item.go (hunk 2 above)
Comment (adambarreiro) [reply vcd/resource_vcd_tm_content_library_item.go]: Removed

File: website/docs/d/tm_content_library_item.html.markdown (hunk 1 above)
Comment (adambarreiro) [reply website/docs/d/tm_content_library_item.html.markdown]: Fixed, thanks for spotting!

**Review by lvirbalas — APPROVED** [review]
This is great.

**Pull request merged**